npm/src/errorhandling/index.ts

```diff
@@ -95,18 +95,30 @@ function hideBrowserCloseMessageAndProgressBar() {
 function showLoggedOutError(login: string) {
   const uploadForm: HTMLFormElement | null =
     document.querySelector("#file-upload-form")
-  const loggedOutError: HTMLDivElement | null = document.querySelector(
+  const progressBarAndMessage: Element | null = document.querySelector(
+    "#progress-bar-and-message"
+  )
+  const loggedOutErrors = document.querySelectorAll(
     ".govuk-error-summary.logged-out-error"
   )
-  const loginLink: HTMLAnchorElement | null = document.querySelector(
+  // eslint-disable-next-line no-undef
```

Comment:
Yes, otherwise the return type`NodeListOf<HTMLAnchorElement>` causes an error. It was the return type that was suggested by IntelliJ, but ESlint doesn't want you using it unless you add this comment above.